pkg/types/args_test.go

```diff
@@ -0,0 +1,98 @@
+package types_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
```

Comment:
This func should still be removed.

You want to end up with this directory structure:
https://github.com/appc/cni/tree/master/pkg/ns

You'll see the suite test file in the above directory holds the `t.testing`

here is what the file looks like in question:
https://github.com/appc/cni/blob/master/pkg/ns/ns_suite_test.go